Epam/Java Basics_UpSkill Lab #1/Strings_and_basics_of_text_processing/Working_with_a_string_as_a_String_or_StringBuilder/src/com/androsor/string_object/HomeTask10.java

```diff
@@ -0,0 +1,22 @@
+package com.androsor.string_object;
+
+import static com.androsor.string_object.IOUtils.enterStringFromConsole;
+
+
+/**
+ * Line X consists of several sentences, each ending with a period, exclamation mark, or question mark.
+ * Determine the number of sentences in line X.
+ */
+public class HomeTask10 {
+
+    public static void main(String[] args) {
+
+        String line = enterStringFromConsole("Введите строку:");
+
+        System.out.printf("Предложенный текст состоит из %d предложений.", countSentence(line));
+    }
+
+    public static int countSentence(String string) {
+        return string.split("[.!?]").length;
```

Comment:
What will happen for such cases like: ". Sentence." and "it's a typical thing when you are using prefix '.test'!"